snaps/learn/tutorials/gas-estimation.md

```diff
@@ -199,10 +242,10 @@ case "hello":
 
 To build and test your Snap:
```

Comment:
```suggestion
Complete the following steps to build and test your Snap:
```

Also, can we change Step 5 to "Build and test _your_ Snap" for consistency?

Reply:
yes, done in both